crates/apub/src/protocol/activities/deletion/delete.rs

```diff
@@ -1,18 +1,29 @@
 use crate::objects::person::ApubPerson;
-use activitystreams::{activity::kind::DeleteType, unparsed::Unparsed};
+use activitystreams::{
+  activity::kind::DeleteType,
+  object::kind::TombstoneType,
+  unparsed::Unparsed,
+};
 use lemmy_apub_lib::object_id::ObjectId;
 use serde::{Deserialize, Serialize};
 use serde_with::skip_serializing_none;
 use url::Url;
 
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Tombstone {
+  pub(crate) id: Url,
+  #[serde(rename = "type")]
+  pub(crate) kind: TombstoneType,
+}
+
```

Comment:
We already had a tombstone struct, in protocol/objects ;)

Reply:
oops